src/server.js

```diff
@@ -1,9 +1,14 @@
 const Hapi = require('hapi');
 const Inert = require('inert');
 const fs = require('fs');
-const path = require('path');
-const socket = require('./sockets.js');
+const Request = require('request');
```

Comment:
Hapi style guide suggests all module variable names start with uppercase, crockford suggests only constructors (used with `new`) should be capitalised. Choose which you are following!